mailbox/scanning-search/src/test/java/org/apache/james/mailbox/store/search/PDFTextExtractor.java

```diff
@@ -28,36 +28,32 @@
 import org.apache.pdfbox.text.PDFTextStripper;
```

Comment:
- It's a fixup, don't introduce new commits when not needed; it makes reviews more complicated to read.
- I'm really not fan of testing tools used only in tests.

Reply:
- Well, I find it interesting to specify the behavior of the objects we are working with.
 - It is also already done for many tests classes (MessageAssert, etc...)
 - Once the tests are written, I don't see what is there cost.

Sorry, I'll do fixup next time.